src/led.py

```diff
@@ -11,131 +11,54 @@
-        self.state = state
-
-    def _animate(self):
-        # TODO(ensonic): refactor or add justification
-        # pylint: disable=too-many-branches
-        while self.running:
-            if self.state:
-                if self.state == 'on':
-                    self.iterator = None
-                    self.sleep = 0.0
-                    self.pwm.ChangeDutyCycle(100)
-                elif self.state == 'off':
-                    self.iterator = None
-                    self.sleep = 0.0
-                    self.pwm.ChangeDutyCycle(0)
-                elif self.state == 'blink':
-                    self.iterator = itertools.cycle([0, 100])
-                    self.sleep = 0.5
-                elif self.state == 'blink-3':
-                    self.iterator = itertools.cycle([0, 100] * 3 + [0, 0])
-                    self.sleep = 0.25
-                elif self.state == 'beacon':
-                    self.iterator = itertools.cycle(
-                        itertools.chain([30] * 100, [100] * 8, range(100, 30, -5)))
-                    self.sleep = 0.05
-                elif self.state == 'beacon-dark':
-                    self.iterator = itertools.cycle(
-                        itertools.chain([0] * 100, range(0, 30, 3), range(30, 0, -3)))
-                    self.sleep = 0.05
-                elif self.state == 'decay':
-                    self.iterator = itertools.cycle(range(100, 0, -2))
-                    self.sleep = 0.05
-                elif self.state == 'pulse-slow':
-                    self.iterator = itertools.cycle(
-                        itertools.chain(range(0, 100, 2), range(100, 0, -2)))
-                    self.sleep = 0.1
-                elif self.state == 'pulse-quick':
-                    self.iterator = itertools.cycle(
-                        itertools.chain(range(0, 100, 5), range(100, 0, -5)))
-                    self.sleep = 0.05
-                else:
-                    logger.warning("unsupported state: %s", self.state)
-                self.state = None
-            if self.iterator:
-                self.pwm.ChangeDutyCycle(next(self.iterator))
-                time.sleep(self.sleep)
-            else:
-                time.sleep(1)
-
-
 def main():
     logging.basicConfig(
         level=logging.INFO,
-        format="[%(asctime)s] %(levelname)s:%(name)s:%(message)s"
-    )
+        format="[%(asctime)s] %(levelname)s:%(name)s:%(message)s")
 
     import configargparse
     parser = configargparse.ArgParser(
-        default_config_files=CONFIG_FILES,
-        description="Status LED daemon")
-    parser.add_argument('-G', '--gpio-pin', default=25, type=int,
-                        help='GPIO pin for the LED (default: 25)')
+        default_config_files=CONFIG_FILES, description="Status LED daemon")
+    parser.add_argument(
+        "-G",
+        "--gpio-pin",
+        default=25,
+        type=int,
+        help="GPIO pin for the LED (default: 25)")
```

Comment:
I did not manually reformat the code, but pyformat did. At some point we have to automate this with some tool, preferably pyformat.